### PR TITLE
Support for InfluxDB 0.9

### PR DESCRIPTION
**Pull request by babilen**
These commits implement support for 0.9, while maintaining backwards compatibility and the ability to still install 0.8 to which it currently defaults.

### CODE REVIEW

File: influxdb/files/influxdb_0.9.sysvinit.jinja [hunk 1]
```diff
@@ -0,0 +1,185 @@
+{% from "influxdb/map.jinja" import influxdb_settings with context -%}
+
+#! /usr/bin/env bash
```
Comment (iggy) [inline influxdb/files/influxdb_0.9.sysvinit.jinja]: Does this end up with a white line above the shebang?

File: pillar.example [hunk 2]
```diff
@@ -1,26 +1,85 @@
...
-  bind-address: '0.0.0.0'
-  logging:
-    directory: '/var/log/influxdb'
-    level: info
-  admin:
-    port: 8083
-  api:
+  # Configuration differs between influxdb versions
+  # 0.8
+>>>>>>> 6cd5894... InfluxDB 0.9 support
```
Comment (iggy) [inline pillar.example]: merge remnant?

**Comment by iggy** [comment]
Looks good other than that.

File: influxdb/files/influxdb_0.9.sysvinit.jinja (hunk 1 above)
Comment (babilen) [inline influxdb/files/influxdb_0.9.sysvinit.jinja]: It does not (hence the `-%}` for whitestpace control)

**Comment by babilen** [comment]
Just addressed the merge issue and tested an installation with both `0.8.8` and `0.9.0-rc25`.

**Comment by iggy** [comment]
Good work. Thanks man.

**Pull request merged**